lib/mix/test/mix/tasks/escript_test.exs

```diff
@@ -65,57 +29,18 @@ defmodule Mix.Tasks.EscriptTest do
     end
   end
 
-  defmodule EscriptErlangMainModule do
-    def project do
-      [
-        app: :escript_test_erlang_main_module,
-        version: "0.0.1",
-        escript: [main_module: :escript_test]
-      ]
-    end
-  end
-
-  defmodule EscriptWithUnknownMainModule do
-    def project do
-      [
-        app: :escript_test_with_unknown_main_module,
-        version: "0.0.1",
-        escript: [main_module: BogusEscriptTest]
-      ]
-    end
-  end
-
-  defmodule EscriptConsolidated do
-    def project do
-      [
-        app: :escript_test_consolidated,
-        build_embedded: true,
```

Comment:
I moved this to the default test, since we expect consolidation to happen by default. Also, I don't think `:build_embedded` is supposed to do anything here.